.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+## What
+
+## Why
```

Comment:
Does it really improve the PR description quality?

A good PR description is concise, yet has the necessary information.   The boilerplate headings make it less concise (the description can implicitly answer "what" and "why" without these headings).  And it may make the quality lower, because the focus becomes only these two points, rather than thinking critically about making a good summary of the PR.

It's a matter of preference, but thank you for considering my opinion.

Reply:
This is just a starting point. Better improvements are welcome. Since most of members are not good at describing PR, even a simplest headings can help them. Also, you can overwrite the template in each repository if you want.

Reply:
One responsibility of the PR reviewer is to check the description.  I expect that between the author and reviewer, a good description can be produced.

Just as often as too little context in the PR description, I've seen the mistake of too much info (especially things that should be comments in the code itself).

The neodm repo used to have a default PR description, but it was a comment about PR guidelines.  But finally we removed it because it was easy to accidentally merge the comment itself.  We moved the guidelines to the README.

https://github.com/groove-x/lovot-neodm?tab=readme-ov-file#prレビューを受ける前の確認項目